### DISCUSSION TITLE
Fresh install using QuickStart guide: {{notification.header}} {{notification.message}} loading

### DISCUSSION THREAD

**Discussion by DodgyBadger** [keycloak/keycloak#11208] [Q&A]
Hello.

I am hoping for some help on this issue which has me stumped. Every time I try to access the admin console, all I get is {{notification.header}} {{notification.message}} loading. This happens on a fresh download in development mode, as instructed on the Quickstart Guide, and in production mode with mariadb. It happens whether accessing from localhost or through a reverse proxy. I am able to access the start page and set the admin user/password, but everything after that returns this error. Here's a short list of what I have tried:

- Searching google for this error brings up a [couple of threads](https://keycloak.discourse.group/t/stuck-on-notification-header-notification-message/725) about resetting the theme in the database. That did not help (the values were already null).
- Messing with permissions on the keycloak folder and running as root.
- Keycloak versions 17.0.01, 17.0.0, and 16.1.1
- openJDK17 and 11 (jre headless)
- Loads of different config options (http-enabled=true, hostname-strict=false, proxy=edge, and more)
- All of this on a different server not running Plesk just in case it had something to do with Plesk's firewall. And then again in a fresh install of Ubuntu Server 20 on a local virtual machine.

The only success I've had was running the Docker version but I abandoned that because of other problems and ultimately too many layers of learning. I would prefer to get this running on bare metal.

Thank you in advance for your help. 🙏

**Comment by csergiu20** [comment]
`chown -R keycloak:keycloak /opt/keycloak`
may solve this.

**Reply by DodgyBadger** [reply]
Thank you for the reply. This wasn't the solution although I think I've figured it out. I've been using lynx to access the admin console on localhost because I'm trying to do this all through SSH on a VPS (probably a key detail I should have included in my original post, but it didn't occur to me it would matter).

As a test, I spun up Ubuntu Desktop in a local virtual machine and everything works as expected accessing localhost:8080/admin in Firefox, but if I try to access through lynx on that same virtual machine, I get the error. My takeaway is that the admin console simply doesn't work with lynx (or any command line browser for that matter). So on to figuring out how to set up access through a reverse proxy.
